### PR TITLE
add: Home Page and Category Page UI

### PR DESCRIPTION
**Pull request by shilpe26**
Added Home Page and Category Page UI with features like:-
▪️Added Router
▪️Navbar
▪️Footer
▪️Responsiveness

Home page  quiz

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
your code looks clean
UI also looks good
PR is ready to merge

**Review by lomesshh — APPROVED** [review]
modular code
all functionalities are working as expected

looks good to merge

**Review by OmKakatkar — APPROVED** [review]
PR looks good

Like:
* Use of variables for colors
* Meaningful variable names
* A video preview for the PR

File: src/pages/landing/Landing.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+						<p className="para-quiz">
+							Test your knowledge about subjects like DBMS, DSA, JAVA, OOPS and
+							more using QuizLet.
+						</p>
+					</div>
+					<div className="btn-quiz mt-32">
+						<Link to="/home">
+							<button
+								type="button"
+								className="btn btn-primary border-12 p-4 w-64"
```
Comment (Varun-28) [inline src/pages/landing/Landing.jsx:22]: - Good use of code formatting
- CSS variables are also used in the right way
- great modular code with small files

**Pull request merged**